lllc/main.cpp

```diff
@@ -118,14 +118,7 @@ int main(int argc, char** argv)
 
 	string src;
 	if (infile.empty())
-	{
-		string s;
-		while (!cin.eof())
-		{
-			getline(cin, s);
-			src.append(s);
```

Comment:
Oh wow, so lll always discarded the newline?

Reply:
Yep